### PR TITLE
design fixes (fonts, icons, cookie management banner)

### PR DESCRIPTION
**Pull request by brindy**
Task/Issue URL: https://app.asana.com/0/414235014887631/1204701159832274/f

and:
* https://app.asana.com/0/414235014887631/1204807921061412/f

Tech Design URL:
CC:

**Description**:

* Remove all references to Proxima Nova in storyboards and xibs
* Updated fonts to set their expected weight using the system font
* Fixed dax logo alignment in address bar
* Fixed dax logo alignment when onboarding
* Fixed cookie management prompt

**Steps to test this PR**:

1. Smoke test the whole app including:
* onboarding
* settings
* address bar
* browsing menu
* anything else you can think of

2. Make sure onboarding animation logo transition looks good on a range of devices

3. Make sure the DDG logo is centre aligned in the address bar

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [ ] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 14
* [ ] iOS 15
* [ ] iOS 16

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/DuckDuckGo/BrowserServicesKit",
         "state": {
           "branch": null,
-          "revision": "2e90e6c20d6f6efe9ce0431ec764475242bee588",
-          "version": "59.1.0"
+          "revision": "5c775d2f5735222e6cd95ca5823ba5b8ccc628d6",
+          "version": "59.1.1"
         }
```
Comment (brindy) [inline DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved]: This wasn't updated on develop.

File: DuckDuckGo/Base.lproj/Settings.storyboard [hunk 2]
```diff
@@ -42,7 +31,7 @@
                                                 <label opaque="NO" multipleTouchEnabled="YES" contentMode="left" text="Set as Default Browser" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" id="xof-5k-PkI">
                                                     <rect key="frame" x="20" y="0.0" width="334" height="43.666667938232422"/>
                                                     <autoresizingMask key="autoresizingMask"/>
-                                                    <fontDescription key="fontDescription" name="ProximaNova-Semibold" family="Proxima Nova" pointSize="16"/>
+                                                    <fontDescription key="fontDescription" type="system" weight="regular" pointSize="16"/>
```
Comment (miasma13) [inline DuckDuckGo/Base.lproj/Settings.storyboard:34]: In multiple cases of `Settings.storyboard` the `ProximaNova-Semibold` is replaced with `type="system" weight="regular"`. Just want to double check if it is on purpose.

**Review by miasma13 — APPROVED** [review]

File: DuckDuckGo/Base.lproj/Settings.storyboard (hunk 2 above)
Comment (brindy) [reply DuckDuckGo/Base.lproj/Settings.storyboard:34]: Yep! Semibold looked really wrong so I went to Figma and it said regular. https://www.figma.com/file/BcyTqJ48TDbFlitfhBq2S5/Settings-(iOS)?type=design&node-id=1702-3347&t=FwgCRTJNj1JFu2F7-0

Comment (miasma13) [reply DuckDuckGo/Base.lproj/Settings.storyboard:34]: That's what I thought. 👍

**Review by miasma13 — APPROVED** [review]

**Pull request merged**